### ISSUE TITLE
Error: Debug Failure. EmptyUri.getFilePath() should not be called

### ISSUE THREAD

**Issue by westernwontons** [microsoft/pyright#6737]
**Describe the bug**
Opening any Python file after updating to version 1.1.340 results in `Error: Debug Failure. EmptyUri.getFilePath() should not be called` and causes `pyright` to crash immediately.

**Code or Screenshots**
Opening an empty Python file is enough to cause the crash.

## LSP Logs

```
[ERROR][2023-12-14 18:28:08] ...lsp/handlers.lua:556	"Notification workspace/didChangeConfiguration defines parameters by name but received parameters by position"
[ERROR][2023-12-14 18:28:08] ...lsp/handlers.lua:556	"Notification workspace/didChangeConfiguration defines 1 params but received 0 arguments"
[ERROR][2023-12-14 18:28:08] .../vim/lsp/rpc.lua:675	"rpc"	"/Users/notimportant/.local/share/nvim/mason/bin/pyright-langserver"	"stderr"	"Error: Debug Failure. EmptyUri.getFilePath() should not be called.\n    at l.getFilePath (/Users/notimportant/.local/share/nvim/mason/packages/pyright/node_modules/pyright/dist/pyright-internal/src/common/uri/emptyUri.ts:58:15)\n    at Function.runWithOptions (/Users/notimportant/.local/share/nvim/mason/packages/pyright/node_modules/pyright/dist/pyright-internal/src/languageService/analyzerServiceExecutor.ts:35:36)\n    at S.updateOptionsAndRestartService (/Users/notimportant/.local/share/nvim/mason/packages/pyright/node_modules/pyright/dist/pyright-internal/src/languageServerBase.ts:525:33)\n    at S.updateSettingsForWorkspace (/Users/notimportant/.local/share/nvim/mason/packages/pyright/node_modules/pyright/dist/pyright-internal/src/languageServerBase.ts:509:14)\n    at async Promise.all (index 0)\n"
```

## Node version
v20.10.0

## Update

It seems that Pyright crashes unconditionally if `pyproject.toml` is not present in the working directory.

**Comment by rchiodo** [comment]
@westernwontons thanks for the issue. Can you reproduce with just the command line? It doesn't seem to repro for me.

**Comment by rchiodo** [comment]
Never mind, I think the callstack is enough information to figure out what the problem is. Thanks.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.341, which I just published.
